scripts/apps/authoring/views/article-edit.html

```diff
@@ -41,8 +41,9 @@
          data-read-only="!_editable"
          data-item="item"
          data-refresh-trigger="refreshTrigger"
-         data-tabindex="editor.headline.order">
-    </div>
+         data-tabindex="editor.headline.order"
```

Comment:
Why different names? `cleanPastedHTML` / `stripPastedStyles`

Reply:
not sure, was using draftjs config name but I guess I can only use that on editor component